spec/helpers/settings_helper_spec.rb

```diff
@@ -0,0 +1,197 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'spec_helper'
+require 'ostruct'
+
+describe SettingsHelper, type: :helper do
+  include Capybara::RSpecMatchers
```

Comment:
@myabc I have no obligation against using Caypbara::RSpecMatchers here if we can avoid having to rewrite what they can provide us. 

A thin wrapper around it might make sense if we are to reuse dom creation testing. But that can be done later once we identify the need.

Regardless of the technical means used, as this is a unit test we should probably be more strict about the expected output. e.g. specify the container dom object ('p'/'li' with classname) we expect the form element to be in.